outputs.tf

```
@@ -1,3 +1,90 @@
+locals {
+  all_data = {
+    for p in var.drupal_projects_list : "${p.project_name}-${p.gitlab_project_id}-${p.release_branch_name}" => {
+      # Add the values you want to store for each project here
+      # Example:
+      namespace          = p.kubernetes_namespace == null ? "${p.project_name}-${p.gitlab_project_id}-${p.release_branch_name}" : p.kubernetes_namespace
+      helm_release_name  = p.helm_release_name == null ? "drupal-${p.release_branch_name}-${p.gitlab_project_id}" : p.helm_release_name
+      bucket_credentials = try(module.drupal_buckets[0].buckets_access_credentials["${p.project_name}-${p.gitlab_project_id}-${p.release_branch_name}-drupal"], null)
+      database_credentials = try(
+        [for cred in module.drupal_databases_and_users[0].sql_users_creds : cred
+      if cred.database == replace("${p.project_name}_${p.gitlab_project_id}_${p.release_branch_name}_dp", "-", "_")][0], null)
+      kubernetes_bucket_secret   = try(local.bucket_secrets_map["${p.project_name}-${p.gitlab_project_id}-${p.release_branch_name}"], null)
+      kubernetes_database_secret = try(local.database_secrets_map["${p.project_name}-${p.gitlab_project_id}-${p.release_branch_name}"], null)
+    }
+  }
+
+  bucket_secrets_map = {
+    for o in local.drupal_buckets_list : replace(o.name, "-drupal", "") => {
+      secret_name = try(
+        kubernetes_secret.bucket_secret_name[o.name].metadata[0].name,
+        null
+      )
+      namespace = try(
+        kubernetes_secret.bucket_secret_name[o.name].metadata[0].namespace,
+        null
+      )
+    }
+  }
+
+  database_secrets_map = {
+    for p in var.drupal_projects_list : "${p.project_name}-${p.gitlab_project_id}-${p.release_branch_name}" => {
+      secret_name = try(
+        kubernetes_secret.database_secret_name[replace("${p.project_name}_${p.gitlab_project_id}_${p.release_branch_name}_dp", "-", "_")].metadata[0].name,
+      null)
+      namespace = try(
+        kubernetes_secret.database_secret_name[replace("${p.project_name}_${p.gitlab_project_id}_${p.release_branch_name}_dp", "-", "_")].metadata[0].namespace,
+        null
+      )
+    }
+  }
+}
+
+
+output "drupal_all_data" {
+  description = "All data for each Drupal project."
+  value       = local.all_data
+}
+
+output "drupal_all_bucket_credentials" {
```

```suggestion
output "drupal_resources_all_bucket_credentials" {
```
or also:

```suggestion
output "drupal_resources_bucket_credentials" {
```

And so on. What do you think?